src/registry/domain/validators/uploaded-package.js

```diff
@@ -21,9 +19,9 @@ module.exports = function(input){
     return returnError('not_valid');
   }
 
-  const file = input[_.keys(input)[0]];
+  const file = input[0];
 
-  if(file.mimetype !== 'application/octet-stream' || !!file.truncated || file.extension !== 'gz' || file.path.indexOf('.tar.gz') < 0){
+  if(file.mimetype !== 'application/octet-stream' || file.truncated || file.filename.indexOf('.tar.gz') < 0){
```

Comment:
Sure. Perhaps we can switch to lodash's contains in a separate PR using this: https://github.com/wix/eslint-plugin-lodash/blob/master/docs/rules/prefer-includes.md

And then move to ES6's when deprecating node 4?